### ISSUE TITLE
Unstable Mission Mode - Rejoining server while shuttle timer is dispatching, creates a new character for that player.

### ISSUE THREAD

**Issue by Kaillera** [Regalis11/Barotrauma#2302]
- [ ] I have searched the issue tracker to check if the issue has already been reported.

**Description**
In Mission Mode if you rejoin a server while the shuttle is counting down to be dispatched, the shuttle will create a clone for that player. You will be able to control both via "control Kaillera [0-1]".

**Version**
~0.9.708

**Comment by Regalis11** [comment]
I don't think this happens any longer?

**Comment by Rokvach** [comment]
I don't think this is possible anymore either and I haven't seen it happen in couple of updates now
